### DISCUSSION TITLE
'Bus Error' while running 'npm run build' or 'npm run dev' in Ubuntu 20.04 server

### DISCUSSION THREAD

**Discussion by azakero** [vercel/next.js#32893] [Help]
Okay so I have been trying to deploy a Next app which fetches data from MongoDB Atlas with the help of Mongoose. Everything seems to be working fine in my local machine but when I try to deploy it in my server, it has been throwing errors.

At first I wasn't even able to install the dependencies. It got stuck and eventually got 'Killed'. Tried a lot of times. Then after installing Next12 I was able to install the dependencies. Then when I try to run the app as dev on my server, I get 'Bus Error'. After that I tried to build the app with 'npm run build' but then it shows the error in the photo below. I don't know what is wrong with it. 

Any idea what could be the reason?

![Screenshot_2](https://user-images.githubusercontent.com/49979539/147661947-18777dd9-c15d-4ae4-b4d9-dafc8efcd0fa.jpg)

**Comment by Xavin616** [comment]
Try deleting node_modules and run npm install.
Worked for me.

**Reply by eciosilva** [reply]
Thank You very very much.

**Reply by hebertcisco** [reply]
Thank You very, very much. Works for me

**Reply by qodlizaka** [reply]
It works for me. Thanks!

**Comment by Okeibunor** [comment]
> Try deleting node_modules and run npm install. Worked for me.

If this doesn't work for you, also delete the package-lock.json file along the node_modules folder.

**Comment by thenurakulathunga** [comment]
Ya it's works..
I'm removed node_modules folder and package-lock.josn both and Run npm i

**Comment by Sajjad-177** [comment]
i am trying to understand why this happens. anyone know why?
